### ISSUE TITLE
Anzeigen des letzten Ladevorgangs 

### ISSUE THREAD

**Issue by alexbln01** [Newan/ioBroker.easee#39]
Hallo, ist es möglich den letzten Ladevorgang noch einzubauen ?
Die geladenen KW und die € des letzten Ladevorgangs.
Mich interessiert nunmal was mich die immer letzte Fahrt gekostet hat, so nehme ich immer den Taschenrechner, das muß doch auch anders gehen :-)

**Comment by Heini1985** [comment]
Hallo alexbln01

ich habe bei mir das Problem mit einem Blockade Script gelöst.
Ich nehme den lifetime Zähler schreibe diesen um 0:00 jeden Tag in eine variable. Dann wenn der lifetime Zähler sich ändert ziehe ich den aktuellen Wert vom gespeicherten ab und dann habe ich mein Tages ggf. Ladungszähler. Mann müsste den Wert nur mit den Stromkosten multiplizieren und dann hat man die Kosten.


Ich bin grade nicht zu Hause aber wenn ich wieder da bin kann ich das Skript hochladen.

Gruß 
Heini1985

**Comment by Heini1985** [comment]
Hier wie versprochen mein skript:

[skript Tageszähler wallbox.txt](https://github.com/Newan/ioBroker.easee/files/7927910/skript.Tageszahler.wallbox.txt)

Die Easee aktualisiert der litime Zähler 1 mal pro stunde.
Mann müsste jetzt den Wert mal kWh preis multiplizieren dann hat man auch die kosten.

Gruß

Heini1985

**Comment by alexbln01** [comment]
@Heini1985 habs eingebaut. Aber wo finde ich den die Debug Ausgabe wo er den Tageszähler in kWh schreibt ?

**Comment by alexbln01** [comment]
Ja, das direkt im Adapter einbauen. Damit man ohne Script arbeiten kann. Was bei mir übrigens nicht funktioniert.
